platform-common/src/main/java/com/softicar/platform/common/network/url/Url.java

```diff
@@ -98,14 +98,21 @@ public String getStartingFromPath() {
 
 	private static String convertToQueryString(Entry<String, List<String>> parameter) {
```

Comment:
changes to this method only aim to beatify parameters with empty value, e.g. instead of `&profiler=` only `&profiler` is added to the URL